lacommunaute/surveys/apps.py

```diff
@@ -32,7 +32,15 @@ def create_recommendations(*args, **kwargs):
                 codename=codename,
                 category=spec["category"],
                 text=spec["text"],
+                service_cats=spec.get("service_cats"),
+                service_subs=spec.get("service_subs"),
             )
         )
-    Recommendation.objects.bulk_create(recommendations, ignore_conflicts=True)
+    for recommendation in Recommendation.objects.bulk_create(recommendations, ignore_conflicts=True):
+        Recommendation.objects.filter(codename=recommendation.codename).update(
+            category=recommendation.category,
+            text=recommendation.text,
+            service_cats=recommendation.service_cats,
+            service_subs=recommendation.service_subs,
+        )
```

Comment:
Est-ce que tu ne pourrais pas plutôt utiliser `update_conflicts` et `update_fields` de https://docs.djangoproject.com/en/5.0/ref/models/querysets/#bulk-create pour faire la même chose sans avoir à le faire à la main ?

Reply:
loving it